ihttp/ihttp.go

```diff
@@ -57,28 +71,84 @@ func (s Handler) Handle(w http.ResponseWriter, req *http.Request) {
 		return
 	}
 	host, port, _ := net.SplitHostPort(req.RemoteAddr)
-	s.Log = s.Log.WithValues("host", host, "port", port)
-	m := path.Dir(req.URL.Path)
-	if strings.HasPrefix(m, "/") {
-		m = trimFirstRune(path.Dir(req.URL.Path))
-	}
+	log := s.Log.WithValues("host", host, "port", port)
 	// If a mac address is provided, log it. Mac address is optional.
-	// Mac address would be useful for logging and tracing.
-	mac, _ := net.ParseMAC(m)
-	s.Log = s.Log.WithValues("mac", mac)
+	mac := parseMac(req.URL.Path)
```

Comment:
Can you rename the variable to `optionalMac`?

Reply:
sorry, not following what value this gives us that we don't already get. I will add a function comment to clarify more about the purpose here.